libs/SmartSync/SmartSync/Classes/Util/SFSyncUpTarget.m

```diff
@@ -71,7 +71,9 @@ + (instancetype)newFromDict:(NSDictionary*)dict {
     }
     // No implementation class - using target type
     else {
-        switch ([self targetTypeFromString:dict[kSFSyncTargetTypeKey]]) {
+        // No target type - assume kSFSyncUpTargetTypeRestStandard (hybrid apps don't specify it a sync up target type by default)
```

Comment:
That bit of logic was lost in https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1892